src/main/java/com/exasol/dbbuilder/dialects/exasol/ExasolImmediateDatabaseObjectWriter.java

```diff
@@ -201,6 +201,11 @@ public void visit(final UdfReturnTypePredefinedEmits emits) {
         }
     }
 
+    @Override
+    public void truncate(final Table table) {
+        writeToObject(table, "DELETE FROM " + table.getFullyQualifiedName());
```

Comment:
```suggestion
        writeToObject(table, "TRUNCATE TABLE " + table.getFullyQualifiedName());
```

Reply:
Exasol also have this: https://docs.exasol.com/sql/truncate.htm

Reply:
oh, I didn't know that. Then I can even move it to the abstract base.